<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>practosuClass</class>
 <widget class="QMainWindow" name="practosuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>practosu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="reloadDb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload osu.db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadMap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeFile">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currentMap">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>osu! Map:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMap">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Manually Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Audio Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Creator:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Version:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Approach Rate:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Circle Size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>HP Drain Rate:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Overall Difficulty:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Speed Multiplier:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QSlider" name="arSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="odSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="csSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hpSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="audioText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="creatorText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="versionText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hpLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="csLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="odLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="arLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>i don't know how to make ui lol</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="practosu.qrc"/>
 </resources>
 <connections/>
</ui>
